### ISSUE TITLE
Ability to monitor by CSS/jQuery selector

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#1]
```
Need to add ability to monitor a page by a selector, similar to how one can 
currently monitor by regex. This entails:

1. Adding a mode (text/regex/selector) variable to each page record in 
localStorage.
2. Updating monitor.js::cleanAndHashPage().
3. Updating monitor.js::addPage() to set the mode field to "text".
4. Updating background.htm::firstRunInit() to correctly set the mode field in 
imported pages.
5. Redesigning the advanced controls in options.htm to accommodate selecting 
from multiple modes (radio group, single textbox for regex/selector).
```

Original issue reported on code.google.com by `max99x` on 10 Feb 2010 at 2:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 16 Feb 2010 at 11:54
* Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Included in v2.1.
```

Original comment by `max99x` on 17 Feb 2010 at 2:05
* Changed state: **Fixed**
